context/deploy.js

```diff
@@ -0,0 +1,21 @@
+//Generated  file 
+'use strict';
+module.exports = {
+    payloadJsonSchema: {
```

Comment:
is it possible to also get other values from the request? e.g. headers etc. Some of the headers will be used for tracing.